include/globals.h

```diff
@@ -95,9 +95,11 @@ typedef struct {
 typedef struct {
   int32_t latitude;
   int32_t longitude;
+#if !(PAYLOAD_OPENSENSEBOX)
   uint8_t satellites;
```

Comment:
Suggest to leave this untouched, doesn't hurt.

Reply:
reworked